packages/video_player/video_player/lib/video_player.dart

```diff
@@ -84,15 +88,15 @@ class VideoPlayerValue {
   /// A description of the error if present.
   ///
   /// If [hasError] is false this is [null].
-  final String errorDescription;
+  final String? errorDescription;
 
   /// The [size] of the currently loaded video.
   ///
   /// Is null when [initialized] is false.
   final Size size;
 
   /// Indicates whether or not the video has been loaded and is ready to play.
-  bool get initialized => duration != null;
+  final bool isInitialized;
```

Comment:
Why did we need to make this name change? Feels like we can make it as non-breaking as possible?
Although I do agree that `isInitialized` is a better name. So your call.

Reply:
Ack.